dialogue-core/src/main/java/com/palantir/dialogue/core/BlacklistingChannel.java

```diff
@@ -166,43 +136,59 @@ public String toString() {
 
     private final class BlacklistState {
```

Comment:
implements `LimitedChannel`

Reply:
I'm not sure that buys us anything, does it?